lib/compat.php

```diff
@@ -131,10 +131,22 @@ function gutenberg_ensure_wp_api_request() {
  * @return array           Filtered settings.
  */
 function gutenberg_disable_editor_settings_wpautop( $settings ) {
-	if ( ! isset( $settings['wpautop'] ) ) {
+	if ( is_array( $settings ) ) {
```

Comment:
I don't think that `$settings` would ever not be an array. What we might want to do here is test first whether the post contains blocks and, if it does, assign `$settings['wpautop']` as explicitly `false` but otherwise do nothing with the key.

Reply:
I like that, a bit cleaner.